test/e2e/spec/test.spec.js

```diff
@@ -0,0 +1,175 @@
+const {test} = require('../browser');
+const {murmur3} = require('murmurhash-js');
+const uuid = require('uuid');
+const {getEmbedStream} = require('../utils/frame');
+const {expect} = require('chai');
+
+describe('Embed Stream', () => {
+
+  it('creates a new asset', test(async (browser, {opts: {url}}) => {
+    const rHash = murmur3(uuid.v4());
```

Comment:
Why hashing the uuid instead of just using the id? Mostly asking because curious.

Reply:
I think I did it because it triggered the same id in the second test I ran 🤔 I wanted to avoid collision at all cost. I think we can remove that now

Reply:
Oh, I remember. It was because `uuid` generates uuids with dashes and for example username does not allow that. I couldn't find a way to overide that easily.